Invalid memory address (clinote user login)

ubuntu 0.50 or 0.60 binary 
running :
clinote user login 

produces : 

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x957911]

goroutine 1 [running]:
github.com/TcM1911/clinote/evernote.NewClient(0xb07ac0, 0xc420172760, 0xc420091a40)
	/home/travis/gopath/src/github.com/TcM1911/clinote/evernote/client_sdk.go:99 +0x2f1
main.defaultClient(0x5ee326)
	/home/travis/gopath/src/github.com/TcM1911/clinote/cmd/helper.go:17 +0xc3
main.glob..func6(0xe515c0, 0xe77500, 0x0, 0x0)
	/home/travis/gopath/src/github.com/TcM1911/clinote/cmd/login.go:33 +0x29
github.com/TcM1911/clinote/vendor/github.com/spf13/cobra.(*Command).execute(0xe515c0, 0xe77500, 0x0, 0x0, 0xe515c0, 0xe77500)
	/home/travis/gopath/src/github.com/TcM1911/clinote/vendor/github.com/spf13/cobra/command.go:636 +0x234
github.com/TcM1911/clinote/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0xe52280, 0xb, 0xa90100, 0xaad4ef)
	/home/travis/gopath/src/github.com/TcM1911/clinote/vendor/github.com/spf13/cobra/command.go:722 +0x2d4
github.com/TcM1911/clinote/vendor/github.com/spf13/cobra.(*Command).Execute(0xe52280, 0xc42007c058, 0x0)
	/home/travis/gopath/src/github.com/TcM1911/clinote/vendor/github.com/spf13/cobra/command.go:681 +0x2b
main.Execute()
	/home/travis/gopath/src/github.com/TcM1911/clinote/cmd/root.go:49 +0x2d
main.main()
	/home/travis/gopath/src/github.com/TcM1911/clinote/cmd/main.go:21 +0x20

--- a/evernote/client_sdk.go
+++ b/evernote/client_sdk.go
@@ -96,7 +96,7 @@ func NewClient(cfg clinote.Configuration) *Client {
 			panic(err.Error())
 		}
 		key = settings.APIKey
-		if settings.Credential.CredType == clinote.EvernoteSandboxCredential {
+		if settings.Credential != nil && settings.Credential.CredType == clinote.EvernoteSandboxCredential {
 			env = ec.SANDBOX
 		}
 	}

